src/main/kotlin/http/achievements/AchievementRoutes.kt

```diff
@@ -0,0 +1,77 @@
+package network.warzone.api.http.achievements
+
+import io.ktor.application.*
+import io.ktor.http.*
+import io.ktor.request.*
+import io.ktor.response.*
+import io.ktor.routing.*
+import network.warzone.api.database.Database
+import network.warzone.api.database.findByIdOrName
+import network.warzone.api.database.models.Achievement
+import network.warzone.api.http.AchievementMissingException
+import network.warzone.api.http.RankMissingException
+import network.warzone.api.http.ValidationException
+import network.warzone.api.util.capitalizeFirst
+import network.warzone.api.util.protected
+import network.warzone.api.util.validate
+import java.util.*
+
+fun Route.addAchievement() {
```

Comment:
To be consistent with the rest of the codebase, could you combine the routes into one function, like in [this file](https://github.com/Warzone/mars-api/blob/master/src/main/kotlin/http/rank/RankRoutes.kt)?